docs/glossary.md

```diff
@@ -1,28 +1,29 @@
 # Glossary
 
-This glossary defines many terms used throughout documentation of Quark.  If
+This glossary defines many terms used throughout documentation of Quark. If
 there is every a concept that seems unclear, check here. This is mainly to
 provide a background and general understanding of the different words and
-concepts that are used.  Other documents will explain in more detail how to
+concepts that are used. Other documents will explain in more detail how to
 combine these concepts to build a particular application.
 
-## Transaction (tx)
+## Transaction
 
 A transaction is a packet of binary data that contains all information to
 validate and perform an action on the blockchain. The only other data that it
 interacts with is the current state of the chain (key-value store), and
-it must have a deterministic action. The tx is the main piece of one request.
+it must have a deterministic action. The transaction is the main piece of one
+request.
 
 We currently make heavy use of [go-wire](https://github.com/tendermint/go-wire)
 and [data](https://github.com/tendermint/go-wire/tree/master/data) to provide
 binary and json encodings and decodings for `struct` or  interface` objects.
 Here, encoding and decoding operations are designed to operate with interfaces
 nested any amount times (like an onion!). There is one public `TxMapper`
-in the basecoin root package, and all modules can register their own transaction types there. This allows us to deserialize the entire tx in
-one location (even with types defined in other repos), to easily embed
-an arbitrary tx inside another without specifying the type, and provide
-an automatic json representation to provide to users (or apps) to
-inspect the chain.
+in the basecoin root package, and all modules can register their own transaction
+types there. This allows us to deserialize the entire transaction in one location
+(even with types defined in other repos), to easily embed an arbitrary tx inside
+another without specifying the type, and provide an automatic json representation
+to provide to users (or apps) to inspect the chain.
```

Comment:
the word 'provide' used too many times... how about "provide an automatic json representation allowing for users (or apps) to inspect the chain."